src/components/article-body/body-components/root.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { ComponentFactory } from '../component-factory';
+import { VerticalSectionSpacing } from '@mdb/flora';
+import { ArticleNode } from '../../../interfaces/article-body-node';
+
+export const Root = ({ children, ...rest }: { children: any }) => {
+    return children.map((child: ArticleNode, i: number) => (
+        <VerticalSectionSpacing key={i} top="xxsmall" bottom="xxsmall">
```

Comment:
Don't use the index as the key as it could cause issues in React https://medium.com/geekculture/reactjs-why-index-as-a-key-is-an-anti-pattern-4b9dc6ef0067